pennylane_lightning/core/_serialize.py

```diff
@@ -347,18 +348,24 @@ def serialize_ops(self, tape: QuantumTape, wires_map: dict = None) -> Tuple[
         uses_stateprep = False
 
         def get_wires(operation, single_op):
-            if operation.name[0:2] == "C(" or operation.name == "MultiControlledX":
-                name = "PauliX" if operation.name == "MultiControlledX" else operation.base.name
-                controlled_wires_list = operation.control_wires
-                if operation.name == "MultiControlledX":
-                    wires_list = list(set(operation.wires) - set(controlled_wires_list))
-                else:
-                    wires_list = operation.target_wires
-                control_values_list = (
-                    [bool(int(i)) for i in operation.hyperparameters["control_values"]]
-                    if operation.name == "MultiControlledX"
-                    else operation.control_values
-                )
+            if isinstance(operation, qml.ops.op_math.Controlled) and not isinstance(
```

Comment:
Great simplification! The previous was quite complicated, with the wires_list depending on MultiControlledX etc. So we didn't need all these if clauses after all?